docker-entrypoint-ssh.sh

```diff
@@ -26,6 +26,13 @@ if [ "$1" = "sshd" ]; then
     fi
     # Execute sshd using absolute path
     shift
+    # if CCACHE_DIR was specified make sure all ssh sessions
+    # inherit the value and turn the cache on with PATH prepend
+    if [ -n "${CCACHE_DIR+x}" ]; then
+        echo "CCACHE_DIR=${CCACHE_DIR}" > /etc/env.d/02distcc-ssh-ccache
```

Comment:
In the ssh case we could in theory not set CCACHE_DIR and not use it as "main switch".

But that would be very confusing to users, because if we do not pass it on to env.d the location actually used in the ssh sessions would be `/home/${SSH_USERNAME}/.cache/ccache`. While whatever value passed to `docker run` would have no effect.

Hence we pass it on to env.d and can also use it as "main switch".